### PR TITLE
Added ability to whisper.

### PR DESCRIPTION
**Pull request by MunWolf**
Closes #6991

You use it by typing /whisper (PlayerName) (Whatever you want to say)

### CODE REVIEW

**Comment by Mailaender** [comment]
I think the duplicate nickname problem is very theoretical and not a blocker here.

**Comment by MicroJOo** [comment]
whispering is indeed a nice idea , specially for FFA ... it will most likely "spice up" the gameplay but at the same time it will also make people mad ... so just keep in mind , this could end up under heavy 	criticism ... 
to avoid that , i'd suggest a settings-option to turn it off ( here it would actually make sense to make a switch) 

and i agree with @Mailaender , duplicate nicknames is not a blocker here

**Comment by Unit158** [comment]
:+1: it works. We could try it out for a playtest and see how it works.

**Comment by MicroJOo** [comment]
"/whisper (PlayerName)"   <--- is a bit long ...

**Comment by pchote** [comment]
"/w&lt;tab&gt; (P&lt;tab&gt;" is pretty short.

**Comment by MicroJOo** [comment]
@pchote : i agree !

File: OpenRA.Game/Network/UnitOrders.cs [hunk 1]
```diff
@@ -91,6 +91,38 @@ public static void ProcessOrder(OrderManager orderManager, World world, int clie
...
+									(client.Name == orderManager.LocalClient.Name || reciever == orderManager.LocalClient.Name))
+									Game.AddChatLine(client.Color.RGB, client.Name + " (whisper)",
+										text.Substring(1 + reciever.Length));
+							}
+							else
+							{
+								var player = world.FindPlayerByClient(client);
+								if (player == null) return;
+
+								if ((world.LocalPlayer != null &&
```
Comment (Mailaender) [inline OpenRA.Game/Network/UnitOrders.cs]: StyleCop violation: `[SA1119] The line contains unnecessary parenthesis.`

**Comment by Mailaender** [comment]
Can you rebase this for additional CI checks please?

**Comment by Mailaender** [comment]
Closing this as it will break our build if merged before a rebase. Come back when you got the time to do it.

**Pull request closed without merging**